rebar.config

```diff
@@ -31,7 +31,8 @@
         {eper, ".*", {git, "git://github.com/basho/eper.git", {tag, "0.78"}}},
         {sext, ".*", {git, "git://github.com/basho/sext.git", {tag, "1.1p3"}}},
         {riak_pipe, ".*", {git, "git://github.com/basho/riak_pipe.git", {branch, "develop-2.2"}}},
-        {riak_api, ".*", {git, "git://github.com/basho/riak_api.git", {branch, "develop-2.2"}}},
         {riak_dt, ".*", {git, "git://github.com/basho/riak_dt.git", {branch, "develop-2.2"}}},
-        {eunit_formatters, ".*", {git, "git://github.com/seancribbs/eunit_formatters", {tag, "0.1.2"}}}
+        {eunit_formatters, ".*", {git, "git://github.com/seancribbs/eunit_formatters", {tag, "0.1.2"}}},
+        {riak_api, ".*", {git, "git://github.com/basho/riak_api.git", {branch, "feature-zl-hll_datatypes"}}},
+        {hyper, ".*", {git, "git://github.com/zeeshanlakhani/hyper", {branch, "wip-zl"}}}
```

Comment:
As per discussion w/ @JeetKunDoug, this should become a tag once https://github.com/basho/hyper/pull/2 is merged.